fenix/app/src/main/res/values/strings.xml

```diff
@@ -378,10 +378,18 @@
     <string name="preferences_default_search_engine">Default search engine</string>
     <!-- Preference for settings related to Search -->
     <string name="preferences_search">Search</string>
-    <!-- Preference for settings related to Search engines -->
+    <!-- Preference for settings related to Search engines  -->
     <string name="preferences_search_engines">Search engines</string>
+    <!-- Preference for settings related to Search engines suggestions-->
+    <string name="preferences_search_engines_suggestions">Suggestions from search engines</string>
     <!-- Preference for settings related to Search address bar -->
     <string name="preferences_search_address_bar">Address bar</string>
+    <!-- Preference for settings related to Search address bar -->
+    <string name="preferences_settings_address_bar">Address bar preferences</string>
```

Comment:
A question and a suggestion! 😊

* It looks like the only place that `preferences_search_address_bar` is used now is the old `search_preferences.xml`. Is it preferred to make a new string, instead of changing the value?
* Let's clarify in this comment (and the one for the Firefox Suggest section below) that this is a preference category, not the preference itself.

Reply:
oh is `search_preferences.xml` no longer used? My first assumption that since there was usage for it elsewhere, i didn't want to disturb it.

Reply:
`search_preferences.xml` isn't currently used, though we could fall back to it through a Nimbus flag if we needed so it isn't dead code quite yet. 

Generally speaking we prefer to create new strings rather than update existing ones. If a string is no longer used, we mark it with the xml tag `moz:RemovedIn="<version>". This avoids errors with reversions, depending on where a patch is in regards to the trains.